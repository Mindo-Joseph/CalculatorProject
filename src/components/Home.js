/* eslint-disable max-len */
import Navbar from './Navbar';

function Home() {
  return (
    <div>

      <Navbar />

      <div className="content">
        <h3>Welcome to our site</h3>
        <div>
          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
        </div>
      </div>

    </div>
  );
}
export default Home;
