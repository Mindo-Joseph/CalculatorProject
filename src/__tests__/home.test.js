import React from 'react';
import '@testing-library/jest-dom/extend-expect';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';

import Home from '../components/Home';

describe('Home', () => {
  test('renders the home component correctly', () => {
    const home = renderer.create(
      <BrowserRouter>
        <Home />
      </BrowserRouter>,
    ).toJSON();
    expect(home).toMatchSnapshot();
  });
});
