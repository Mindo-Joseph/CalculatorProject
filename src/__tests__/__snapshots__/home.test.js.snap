// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders the home component correctly 1`] = `
<div>
  <div
    className="header"
  >
    <h1>
      Math Magicians
    </h1>
    <div
      className="navbar"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        Home 
      </a>
      <a
        href="/calculator"
        onClick={[Function]}
      >
        Calculator 
      </a>
      <a
        href="/quote"
        onClick={[Function]}
      >
        Quote 
      </a>
    </div>
  </div>
  <div
    className="content"
  >
    <h3>
      Welcome to our site
    </h3>
    <div>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
      </p>
    </div>
  </div>
</div>
`;
